### ISSUE TITLE
minigun quick: Phase undefined started.

### ISSUE THREAD

**Issue by tomgco** [shoreditch-ops/minigun#28]
```
$ minigun quick -d 600 -r 200 -o out-`date +"%s"` http://127.0.0.1:8124
Log file: out-1450455607.json
Phase undefined started - duration: 600s
```

```
$ minigun -V
1.2.14
```

**Comment by hassy** [comment]
:+1: Known issue, fix incoming.

![soon.js](http://i.giphy.com/fZ3Q3qRYCw0Zq.gif)

**Comment by hassy** [comment]
Fixed in `1.2.6`
